resources/dicts/relationship_events/normal_interactions/platonic/increase.json

```diff
@@ -175,6 +236,98 @@
 			"leader"
 		]
 	},
+	{
+		"id": "platonic_inc_med13",
+		"interactions": [
+			"m_c, after seeing r_c down in the dumps on the other side of camp, bounded over and started messing around trying to cheer {PRONOUN/r_c/object} up."
+		],
+		"main_trait_constraint": [
+			"troublesome",
+            "arrogant",
+            "childish",
+            "competitive",
+            "daring",
+            "fierce",
+            "flamboyant",
+            "playful",
+            "rebellious",
+            "sincere",
+            "attention-seeker",
+            "bouncy",
+            "impulsive",
+            "sweet"
+		]
+	},
+    {
+		"id": "platonic_inc_med14",
+		"interactions": [
+			"m_c, after seeing r_c down in the dumps on the other side of camp, settled beside {PRONOUN/m_c/object} offering a silent comfort that spoke louder than words."
+		],
+		"random_trait_constraint": [
+		    "calm",
+            "cold",
+            "compassionate",
+            "gloomy",
+            "grumpy",
+            "lonesome",
+            "loving",
+            "loyal",
+            "nervous",
+            "sincere",
+            "thoughtful",
+            "wise",
+            "polite",
+            "quiet",
+            "sweet"
+		]
+	},
+	{
+		"id": "platonic_inc_med15",
+		"interactions": [
+            "m_c and r_c mplayed around during patrol and got in a bit of trouble but still returned to camp smiling.",
```

Comment:
```suggestion
            "m_c and r_c mplayed around during patrol and got in a bit of trouble, but still returned to camp smiling.",
```

Reply:
Fixed and also fixed a typo we both missed in that one! Lmao